packs_edit.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+namespace Nottingham\PackManagement;
+
+
+// Get the pack category.
+$queryCategory = $module->query( 'SELECT ems.`value` FROM redcap_external_module_settings ems ' .
```

Comment:
Pretty much the same SQL query does appear elsewhere (except with order by clause).  Could it be put in a separate function call?